### PR TITLE
private itemtracking done (dynamic dictonary)

### PR DESCRIPTION
**Pull request by MShadiF**

### CODE REVIEW

**Review by DeeKahy — COMMENTED** [review]
looks good but need @mantarias to look at this

File: public/itemtracking/stuff.js
Comment (mantarias) [inline public/itemtracking/stuff.js:1]: Has some bugs
add item global suggestions doesn't work
if you click add item type something in the name field, close it, and go to create new item you won't be able to type anything in the name field

**Review by mantarias — CHANGES_REQUESTED** [review]
Has some bugs
add item global suggestions doesn't work
if you click add item and type something in the name field, close it, and go to create new item you won't be able to type anything in the name field

File: public/itemtracking/stuff.js
Comment (mantarias) [reply public/itemtracking/stuff.js:1]: Seems CSS is fucked their location is different from what you see you can focus on developer tools and be able to type but when you hover over them, they are somewhere else

**Pull request closed without merging**